recipes/mqc/build.sh

```diff
@@ -0,0 +1 @@
+#! /bin/bash
```

Comment:
As the package exists of one main perl script and a folder with some extra dependency scripts in python and R, I suppose I need to copy all those scripts here to $PREFIX?